internal/config/config.go

```diff
@@ -43,49 +44,116 @@ type Jobs struct {
 }
 
 type Scenarios struct {
-	HighVolumeWrites HighVolumeWrites `yaml:"highVolumeWrites"`
-	HighVolumeReads  HighVolumeReads  `yaml:"highVolumeReads"`
+	HighVolumeWrites *HighVolumeWrites `yaml:"highVolumeWrites,omitempty"`
+	HighVolumeReads  *HighVolumeReads  `yaml:"highVolumeReads,omitempty"`
+}
+
+func (s *Scenarios) IsWriteTestRunnable() bool {
+	if s == nil {
+		return false
+	}
+
+	if s.HighVolumeWrites == nil {
+		return false
+	}
+
+	return s.HighVolumeWrites.Enabled
+}
+
+func (s *Scenarios) IsReadTestRunnable() bool {
+	if s == nil {
+		return false
+	}
+
+	if s.HighVolumeReads == nil {
+		return false
+	}
+
+	return s.HighVolumeReads.Enabled
 }
```

Comment:
nit. We can continue using the suffix `Enabled` for these methods, wdyt? `Enabled` implies a simply flag, while `Runnable` might imply for the user that we check more than a simple on/off switch, e.g. goroutines etc.